.travis.yml

```diff
@@ -36,8 +36,7 @@ before_deploy:
     - cd gui
```

Comment:
You can just issue the following command:
```
python gui/build.py bundle
```
Because the uic and rcc commands are done anyway when you use the bundle command (https://github.com/mozilla/mozregression/blob/master/gui/build.py#L75). That will allow you to remove the "cd " commands also.